ex2/ex2.c

```diff
@@ -5,10 +5,26 @@
 #include <stdio.h>
 #include <unistd.h>
 #include <stdlib.h>
+#include <time.h>
+#include <sys/wait.h>
+
 
 int main(void)
 {
     // Your code here 
-    
+FILE * fp;
+pid_t pid;
+time_t now;
+time(&now);
+
+   fp = fopen ("text.txt", "a");
+   fprintf(fp,"Tody is : %s", ctime(&now));
+   fprintf(fp, "%s %s %s %d\n", "parent PID written", "to", "on Nov 11", 2018);
+   pid = fork();
+   wait(100);
```

Comment:
Your program is working but this line is giving me warnings. I believe that the wait() function takes a pointer as its argument, and you are putting an integer in here, which is making my compiler unhappy. I don't know if it'll still work if you put NULL in there but it might be worth a little noodling around after you're done the assignment because it's good to know how wait() works.